src/components/home/Home.jsx

```diff
@@ -23,11 +23,6 @@ const ListImg = ({ src }) => (
 
 
 class Home extends Component {
-  componentWillMount() {
-    const { getLimitPosts } = this.props;
-    getLimitPosts();
-  }
```

Comment:
Remove this because we already request the posts at indexPage.
PS. we should use componentDidMount instead of componentWillMount.